pkg/controllers/testutil/zcmd/zpool/fake.go

```diff
@@ -0,0 +1,67 @@
+/*
+Copyright 2020 The OpenEBS Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package zpool
+
+import (
+	internalapi "github.com/openebs/api/pkg/internalapis/apis/cstor"
```

Comment:
Internal api can not be used. This is just a helper hub for conversion server. 
Please use the latest API.

Reply:
We don't have pool topology information(since it is internal helper structure for the pool) in the latest API (or) We should move out from API repo snd add in cstor-operators repo. Open for discussion.